ertviz/assets/ert-style.json

```diff
@@ -107,7 +107,7 @@
             "margin": {
                 "l": 5,
                 "b": 5,
-                "t": 5,
+                "t": 25,
```

Comment:
So we want to increase margin?

Reply:
yes the menu was covering the top plot in the legend